AD419.DataHelper.Web/Models/AccountWithDifferentTotal.cs

```diff
@@ -16,30 +16,10 @@ public partial class AccountWithDifferentTotal
         [Display(Name = "FFY Expenses By ARC Total")]
         public decimal FfyExpensesByArcTotal { get; set; }
 
-        private string _expensesTotal;
         [Display(Name = "Expenses Total")]
-        public string ExpensesTotal
-        {
-            get
-            {
-                return string.IsNullOrEmpty(_expensesTotal) ? "Account did not get transferred to AD-419 expenses table" : _expensesTotal;
-            }
-            set
-            {
-                _expensesTotal = value;
-            }
-        }
+        public decimal? ExpensesTotal { get; set; }
 
         [NotMapped]
-        public string Difference
-        {
-            get
-            {
-                if (_expensesTotal != null)
-                    return (FfyExpensesByArcTotal - Convert.ToDecimal(_expensesTotal)).ToString(CultureInfo.InvariantCulture);
-
-                return "N/A";
-            }
-        }
+        public string Difference => ExpensesTotal != null ? (FfyExpensesByArcTotal - Convert.ToDecimal(ExpensesTotal)).ToString(CultureInfo.InvariantCulture) : "N/A";

```

Comment:
This works but when it's a nullable type you usually just do `HasValue`, as in `ExpensesTotal.HasValue ? ...`